config/index.js

```diff
@@ -797,6 +797,12 @@ var conf = convict({
       format: 'nat',
       env: 'TOTP_STEP_SIZE'
     },
+    window: {
+      doc: 'Tokens in the previous x-windows that should be considered valid',
+      default: 1,
+      format: 'nat',
+      env: 'TOTP_CODE_WINDOw'
```

Comment:
typo `TOTP_CODE_WINDOw`

Reply:
nit: can we name this `totpWindow` ? 

1. env var is called `..._CODE_WINDOW`
(2. and also `window` is a browser global which is a bit trippy)